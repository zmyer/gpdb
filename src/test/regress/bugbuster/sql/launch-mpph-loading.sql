\! psql -d template1 -f ./sql/mpph_load_heap.sql

\! psql -d template1 -f ./sql/mpph_load_AO.sql

\! psql -d template1 -f ./sql/mpph_load_CO.sql
