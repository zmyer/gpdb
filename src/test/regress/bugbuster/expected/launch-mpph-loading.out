\! psql -d template1 -f ./sql/mpph_load_heap.sql
DROP DATABASE
CREATE DATABASE
You are now connected to database "mpph_heap" as user "simiob".
psql:./sql/mpph_load_heap.sql:6: NOTICE:  table "customer" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_heap.sql:7: NOTICE:  table "lineitem" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_heap.sql:8: NOTICE:  table "nation" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_heap.sql:9: NOTICE:  table "orders" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_heap.sql:10: NOTICE:  table "part" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_heap.sql:11: NOTICE:  table "partsupp" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_heap.sql:12: NOTICE:  table "region" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_heap.sql:13: NOTICE:  table "supplier" does not exist, skipping
DROP TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
\! psql -d template1 -f ./sql/mpph_load_AO.sql
DROP DATABASE
CREATE DATABASE
You are now connected to database "mpph_ao" as user "simiob".
psql:./sql/mpph_load_AO.sql:4: NOTICE:  table "customer" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_AO.sql:5: NOTICE:  table "lineitem" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_AO.sql:6: NOTICE:  table "nation" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_AO.sql:7: NOTICE:  table "orders" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_AO.sql:8: NOTICE:  table "part" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_AO.sql:9: NOTICE:  table "partsupp" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_AO.sql:10: NOTICE:  table "region" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_AO.sql:11: NOTICE:  table "supplier" does not exist, skipping
DROP TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
\! psql -d template1 -f ./sql/mpph_load_CO.sql
DROP DATABASE
CREATE DATABASE
You are now connected to database "mpph_co" as user "simiob".
psql:./sql/mpph_load_CO.sql:4: NOTICE:  table "customer" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_CO.sql:5: NOTICE:  table "lineitem" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_CO.sql:6: NOTICE:  table "nation" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_CO.sql:7: NOTICE:  table "orders" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_CO.sql:8: NOTICE:  table "part" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_CO.sql:9: NOTICE:  table "partsupp" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_CO.sql:10: NOTICE:  table "region" does not exist, skipping
DROP TABLE
psql:./sql/mpph_load_CO.sql:11: NOTICE:  table "supplier" does not exist, skipping
DROP TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
CREATE TABLE
